setup.py

```diff
@@ -118,6 +118,8 @@ def __init__(self):
             includes.append(os.path.join(os.getenv('PROTOBUF_INCDIR')))
         elif use_conda:
             includes.append(os.path.join(os.getenv('CONDA_PREFIX'), "Library", "Include"))
+        else:
+            print("Warning: Environment Variable PROTOBUF_INCDIR or CONDA_PREFIX is not set, which may cause protobuf including folder error.")
```

Comment:
Thanks! "conda_prefix" is only needed when "use_conda" is true. Move the warning message into "elif use_conda" block may be better.

Reply:
use_conda = os.getenv('CONDA_PREFIX') and platform.system() == 'Windows'; So the env variable determines use_conda. Here is better?

Reply:
:), Makes sense.

Reply:
Shall we error out the setup here?

Shall we make it explicitly that users should set PROTOBUF_INCDIR or CONDA_PREFIX?